src/Migrations/M022UpdatePostRevisions.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Update the limit of post revisions.
+ */
+class M022UpdatePostRevisions extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        global $wpdb;
+
+        if (!get_option('revisions_to_keep')) {
```

Comment:
We can make it little less code, if we make it -
```
        $MAX_REVISIONS = get_option('revisions_to_keep');
        if (!$MAX_REVISIONS) {
            // If not, set 20 revisions by default
            $MAX_REVISIONS = 20;
            update_option('revisions_to_keep', $MAX_REVISIONS);
        }
```
This way we will call `get_option()` function once.

Reply:
You are correct!